resources/seccomp/aarch64-unknown-linux-musl.json

```diff
@@ -516,6 +559,19 @@
                     }
                 ]
             },
+            {
+                "syscall": "tkill",
```

Comment:
I didn't notice this syscall in the previous PR. Is this new ? What threads can someone affect using this syscall ? Only the ones started inside the jailer ?

Reply:
Yes, it is for sending signals to a `tid`, so it's only for threads inside the jailer. See [man page](https://man7.org/linux/man-pages/man2/tkill.2.html) for clarifications.